src/modules/site-deal/site-deal.module.ts

```diff
@@ -2,14 +2,14 @@ import { Module } from '@nestjs/common';
 import { CustodianModule } from '@ukef/modules/custodian/custodian.module';
 import { SharepointModule } from '@ukef/modules/sharepoint/sharepoint.module';
 
-import { DealFolderService } from './deal-folder.service';
+import { DealFolderCreationService } from './deal-folder-creation.service';
+import { FacilityFolderCreationService } from './facility-folder-creation.service';
+import { SiteDealNotFoundExceptionToBadRequestTransformInterceptor } from './interceptor/site-deal-not-found-exception-to-bad-request-transform.interceptor';
 import { SiteDealController } from './site-deal.controller';
-import { SiteDealService } from './site-deal.service';
-import { SiteDealNotFoundExceptionToBadRequestTransformInterceptor } from './site-deal-not-found-exception-to-bad-request-transform.interceptor';
 
 @Module({
   imports: [CustodianModule, SharepointModule],
   controllers: [SiteDealController],
-  providers: [SiteDealService, SiteDealNotFoundExceptionToBadRequestTransformInterceptor, DealFolderService],
+  providers: [FacilityFolderCreationService, SiteDealNotFoundExceptionToBadRequestTransformInterceptor, DealFolderCreationService],
```

Comment:
Does `SiteDealNotFoundExceptionToBadRequestTransformInterceptor` need to be a provider?